setup.sh

```diff
@@ -12,14 +12,19 @@ if [ ! -e ${ADAFRUIT_DIR} ]; then
 	exit 2
 fi
 
-apt-get install -y python python-dev python-pip python-smbus lighttpd
+apt-get install -y python python-dev python-pip python-smbus nginx uwsgi uwsgi-plugin-python
 pip install flask
 
 pushd ${ADAFRUIT_DIR} > /dev/null
 python setup.py install
 popd > /dev/null
 
-cp www/lighttpd.conf /etc/lighttpd/lighttpd.conf
+rm -rf /var/www/rovercode > /dev/null
+mkdir /var/www/rovercode
```

Comment:
I would suggest adding the `-p` option to `mkdir` so that it will work if `www` isn't there also